### PR TITLE
Configuration option to enable/disable CPU JIT optimizations

### PR DESCRIPTION
**Pull request by MerryMage**
For debugging purposes, CPU optimizations can be disabled.

### CODE REVIEW

**Review by sunho — DISMISSED** [review]

File: vita3k/gui/src/settings_dialog.cpp [hunk 1]
```diff
@@ -251,6 +259,12 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Select your preferred cpu backend.");
         ImGui::Spacing();
+        if (config.cpu_backend == "Dynarmic") {
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:262]: Is good but maybe hetter with using enum here
```suggestion
        if (config_cpu_backend == CpuBackend:: Dynarmic) {
```

File: vita3k/gui/src/settings_dialog.cpp [hunk 2]
```diff
@@ -135,6 +135,7 @@ static bool get_custom_config(GuiState &gui, HostState &host, const std::string
             if (!custum_config_xml.child("core").empty()) {
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:135]:
```suggestion
            if (!custom_config_xml.child("core").empty()) {
```
Please 🤦‍♂️ my fail

File: vita3k/gui/src/settings_dialog.cpp [hunk 3]
```diff
@@ -135,6 +135,7 @@ static bool get_custom_config(GuiState &gui, HostState &host, const std::string
...
                 const auto core_child = custum_config_xml.child("core");
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:136]:
```suggestion
                const auto core_child = custom_config_xml.child("core");
```
Same

**Comment by Zangetsu38** [comment]
If you can set two last commit nside first for config please 😁

File: vita3k/gui/src/settings_dialog.cpp [hunk 4]
```diff
@@ -251,6 +259,12 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Select your preferred cpu backend.");
         ImGui::Spacing();
+        if (config_cpu_backend == CpuBackend::Dynarmic) {
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings_dialog.cpp:262]:
```suggestion
        if (config_cpu_backend == CPUBackend::Dynarmic) {
```
Sorry i review on my tablette lol i have forget is writed like that

**Review by Zangetsu38 — APPROVED** [review]
Perfect

**Pull request merged**